packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -23,13 +23,20 @@ import {SERVER, CLIENT_OPTIONAL, REQUEST_PROCESSOR} from '../../configs/webpack/
 
 const projectDir = process.cwd()
 const projectWebpackPath = path.resolve(projectDir, 'webpack.config.js')
+const projectPkgPath = path.resolve(projectDir, 'package.json')
+const projectPkg = require(projectPkgPath)
 
 const chalk = require('chalk')
 
 const CONTENT_TYPE = 'content-type'
 const CONTENT_ENCODING = 'content-encoding'
 const NO_CACHE = 'max-age=0, nocache, nostore, must-revalidate'
 
+/**
+ * @private
+ */
+export const PATH_TO_LOADING_SCREEN = `/__mrt/loading-screen/index.html?loading=1&project=${projectPkg.name}`
```

Comment:
Using the name `project` might imply to users that they can change the project name here to load a different project, even though they can't. Or maybe they won't think that, I dunno. 🤷